docs/source/3x/PT_WeightOnlyQuant.md

```diff
@@ -14,21 +16,68 @@ PyTorch Weight Only Quantization
+| Algorithms/Backend |   PyTorch eager mode  |  
+|--------------|----------|
+|       RTN      |  &#10004;  |
+|       GPTQ     |  &#10004;  |
+|       AutoRound|  &#10004;  |
+|       AWQ      |  &#10004;  |
+|       TEQ      |  &#10004;  | 
+|       HQQ      |  &#10004;  |
+> **RTN:** A quantification method that we can think of very intuitively. It does not require additional datasets and is a very fast quantization method. Generally speaking, RTN will convert the weight into a uniformly distributed integer data type, but some algorithms, such as Qlora, propose a non-uniform NF4 data type and prove its theoretical optimality.
 
+> **GPTQ:** A new one-shot weight quantization method based on approximate second-order information, that is both highly-accurate and highly efficient[4]. The weights of each column are updated based on the fixed-scale pseudo-quantization error and the inverse of the Hessian matrix calculated from the activations. The updated columns sharing the same scale may generate a new max/min value, so the scale needs to be saved for restoration.
+
+> **AutoRound:** AutoRound is an advanced weight-only quantization algorithm for low-bits LLM inference. It's tailored for a wide range of models and consistently delivers noticeable improvements, often significantly outperforming SignRound[6] with the cost of more tuning time for quantization.
+
+> **AWQ:** Proved that protecting only 1% of salient weights can greatly reduce quantization error. the salient weight channels are selected by observing the distribution of activation and weight per channel. The salient weights are also quantized after multiplying a big scale factor before quantization for preserving.
+
+> **TEQ:** A trainable equivalent transformation that preserves the FP32 precision in weight-only quantization. It is inspired by AWQ while providing a new solution to search for the optimal per-channel scaling factor between activations and weights.
+
+> **HQQ:** The HQQ[7] method focuses specifically on minimizing errors in the weights rather than the layer activation. Additionally, by incorporating a sparsity-promoting loss, such as the $l_{p<1}$-norm, we effectively model outliers through a hyper-Laplacian distribution. This distribution more accurately captures the heavy-tailed nature of outlier errors compared to the squared error, resulting in a more nuanced representation of error distribution.
 
 ## Usage
 
 ### Get Started
 
-The INC 3x New API supports quantizing PyTorch models using prepare and convert for WeightOnlyQuant quantization.
+The INC 3x New API supports quantizing PyTorch models using prepare and convert [APIs](./PyTorch.md#quantization-apis) for WeightOnlyQuant quantization.
+
+#### Commom arguments
+| Config | Capability |
+|---|---|
+| dtype (str)| ['int', 'nf4', 'fp4'] |
+| bits (int)| [1, ..., 8] |
+| group_size (int)| [-1, 1, ..., $C_{in}$] |
+| use_sym (bool)| [True, False] |
```

Comment:
we can add double_quant args here and note that only RTN&GPTQ support it.

Reply:
Thanks, added https://github.com/intel/neural-compressor/pull/1824/commits/86f99577899704cbf0d614502cd7b6f5b21ca17e